lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUBroadcastReduceWithFilePartitionDataSetTest.cs

```diff
@@ -0,0 +1,360 @@
+                cm = cm.Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.FilePathForPartitions, sourceFilePath + i);
+            }
+            
+            return cm.Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.CopyToLocal, "true")
+                     .Set(FileSystemInputPartitionConfiguration<IEnumerable<Row>>.FileSerializerConfig, GetRowSerializerConfigString())
+                     .Build();
+        }
+
+        /// <summary>
+        /// Create test data as bytes
+        /// </summary>
+        /// <param name="dataNumber"></param>
+        /// <returns></returns>
+        private static byte[] CreateTestData(int dataNumber)
+        {
+            var bytes = new byte[dataNumber];
+            for (int i = 0; i < dataNumber; i++)
+            {
+                bytes[i] = (byte)i;
+            }
+            return bytes;
+        }
+
+        /// <summary>
+        /// Create a test data file as input file
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <param name="bytes"></param>
+        private static void MakeLocalTestFile(string filePath, byte[] bytes)
+        {
+            if (File.Exists(filePath))
+            {
+                File.Delete(filePath);
+            }
+
+            using (var s = File.Create(filePath))
+            {
+                foreach (var b in bytes)
+                {
+                    s.WriteByte(b);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Test DeSerializer
+        /// </summary>
+        private class RowSerializer : IFileDeSerializer<IEnumerable<Row>>
+        {
+            [Inject]
+            private RowSerializer()
+            {
+            }
+
+            /// <summary>
+            /// read all the files in the set and return byte read one by one
+            /// </summary>
+            /// <param name="filePaths"></param>
+            /// <returns></returns>
+            public IEnumerable<Row> Deserialize(ISet<string> filePaths)
+            {
+                foreach (var f in filePaths)
+                {
+                    using (FileStream stream = File.Open(f, FileMode.Open))
+                    {
+                        BinaryReader reader = new BinaryReader(stream);
+                        while (reader.PeekChar() != -1)
+                        {
+                            yield return new Row(reader.ReadByte());
+                        }
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Build RowSerialzer configuration and serialize it as string so that to pass it to driver
+        /// </summary>
+        /// <returns></returns>
+        private static string GetRowSerializerConfigString()
+        {
+            var serializerConf = TangFactory.GetTang().NewConfigurationBuilder()
+                .BindImplementation<IFileDeSerializer<IEnumerable<Row>>, RowSerializer>(
```

Comment:
`<IFileDeSerializer<IEnumerable<Row>>, RowSerializer>` can be omitted (we use `BindImplementation` without `<>` more frequently).